index.html

```diff
@@ -0,0 +1,321 @@
+        position: absolute;
+        font-family: sans-serif;
+        width: 19%;
+      }
+
+      #skills {
+        margin-left: 49%;
+        margin-top: 20%;
+        position: absolute;
+      }
+
+      #skills1 {
+        margin-left: 49%;
+        margin-top: 22%;
+        position: absolute;
+        font-family: sans-serif;
+      }
+
+      ul {
+        list-style: none;
+        padding-left: 0;
+      }
+
+      #languages {
+        margin-left: 70%;
+        margin-top: 40%;
+        position: absolute;
+      }
+
+      #languageslist {
+        margin-left: 70%;
+        margin-top: 42%;
+        position: absolute;
+        font-family: sans-serif;
+      }
+
+      #exhibitions {
+        margin-left: 70%;
+        margin-top: 8%;
+        position: absolute;
+        width: 15%;
+      }
+
+      #exhibitionslist {
+        margin-left: 70%;
+        margin-top: 10%;
+        position: absolute;
+        font-family: sans-serif;
+        width: 25%;
+        line-height: 20px;
+      }
+
+      #extracurriculars {
+        margin-left: 70%;
+        margin-top: 22%;
+        position: absolute;
+        width: 15%;
+      }
+
+      #extracurricularslist {
+        margin-left: 70%;
+        margin-top: 24%;
+        position: absolute;
+        font-family: sans-serif;
+        width: 16%;
+      }
+
+      #reference {
+        margin-left: 49%;
+        margin-top: 43%;
+        position: absolute;
+        /*font-family: sans-serif;
+            font-weight: 700;
+            letter-spacing: 3px;*/
+      }
+
+      #referencelist {
+        margin-left: 49%;
+        margin-top: 45%;
+        position: absolute;
+        font-family: sans-serif;
+      }
+
+      a {
+        color: #323232;
+        text-decoration-line: none;
+      }
+
+      #indent {
+        text-indent: 50px;
+        text-align: unset;
+      }
+      
+      #in1 {        
+        margin-left: 12%;        
+      }
+      
+      #in2 {        
+        margin-left: 12%;        
+      }
+      
+      
+    </style>
+  </head>
+
+  <body>
+    <h1 id="name">SAMMY AL-ASMAR</h1>
+    <p id="contact">
+      (615)-967-7362<br />
+      samuelalasmar@gmail.com<br />
+      <a href="http://sammyalasmar.com/" target="_blank">sammyalasmar.com</a>
+    </p>
+
+    <p id="address">
+      708 St. Paul, Apt 8<br />
+      Baltimore, MD-21202
+    </p>
+
+    <h2 id="objective">About</h2>
+    <p id="p1">
+      I am actively seeking a graphic design internship for the summer of 2020
+      that would allow me to experiment with type and image to create ambitious
+      and meaningful design with a team.
+    </p>
+
+    <h2 id="experience">Experience</h2>
+    <p id="p2">
+      <strong>MICA GD Department Intern</strong><br/>
+      Baltimore, MD<br />
+      <span id="in1">Aug 2019–Present<br/></span>
+      Design materials for department events and GD brand. Assist in organizing
+      MICA facilities for department and students.
+    </p>
```

Comment:
The `br` element generally should be reserved for meaningful line breaks, like in an address -- as on line 228 -- or poem; see [MDN's docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br) for more info. Here, it's doing too much work and is not meaningful; it's actually just filling the role of other, more semantic HTML. Consider instead:


```suggestion
    <h3>MICA GD Department Intern</h3>
    <p>Baltimore, MD</p>
    <p>Aug 2019–Present</p>
    <p>
      Design materials for department events and GD brand. Assist in organizing
      MICA facilities for department and students.
    </p>
```